src/Env/Defaults/Defaults.php

```diff
@@ -9,27 +9,27 @@
  * file that was distributed with this source code.
  */
 
-namespace Manala\Manalize\Env\Metadata;
+namespace Manala\Manalize\Env\Defaults;
 
 /**
- * A metadata bag containing all defaults for a given env type.
+ * A bag containing all defaults for a given env.
  *
  * @author Robin Chalas <robin.chalas@gmail.com>
  */
-final class MetadataBag
+final class Defaults
```

Comment:
The motivation is that we need to make a clear distinction between the project's metadata and the env defaults, it's too much confusing to have the same naming for both. About the choice of `defaults`, it just sounds logical to me as the file present in the env contains its defaults. I'm quite surprised by your comment but let's see if we can find a better one or another way to make this less confusing, it started to become harder to keep this separation clear enough.